wdl2cwl/tests/oop_cwl_files/CollectReadgroupBamQualityMetrics.cwl

```diff
@@ -0,0 +1,72 @@
+class: CommandLineTool
+id: CollectReadgroupBamQualityMetrics
+inputs:
+  - id: input_bam
+    type: File
+  - id: input_bam_index
+    type: File
+  - id: output_bam_prefix
+    type: string
+  - id: ref_dict
+    type: File
+  - id: ref_fasta
+    type: File
+  - id: ref_fasta_index
+    type: File
+  - id: collect_gc_bias_metrics
+    default: true
+    type: boolean
+  - id: preemptible_tries
+    type: int
+outputs:
+  - id: alignment_summary_metrics
+    type: File
+    outputBinding:
+        glob: $(inputs.output_bam_prefix + '.alignment_summary_metrics')
+  - id: gc_bias_detail_metrics
+    type: File
+    outputBinding:
+        glob: $(inputs.output_bam_prefix + '.gc_bias.detail_metrics')
+  - id: gc_bias_pdf
+    type: File
+    outputBinding:
+        glob: $(inputs.output_bam_prefix + '.gc_bias.pdf')
+  - id: gc_bias_summary_metrics
+    type: File
+    outputBinding:
+        glob: $(inputs.output_bam_prefix + '.gc_bias.summary_metrics')
+requirements:
+  - class: DockerRequirement
+    dockerPull: us.gcr.io/broad-gotc-prod/picard-cloud:2.23.8
+  - class: InitialWorkDirRequirement
+    listing:
+      - entryname: example.sh
+        entry: |4
+
+            # These are optionally generated, but need to exist for Cromwell's sake
+            touch $(inputs.output_bam_prefix).gc_bias.detail_metrics \
+            $(inputs.output_bam_prefix).gc_bias.pdf \
+            $(inputs.output_bam_prefix).gc_bias.summary_metrics
+
+            java -Xms5000m -jar /usr/picard/picard.jar \
+              CollectMultipleMetrics \
+              INPUT=$(inputs.input_bam.path) \
+            REFERENCE_SEQUENCE=$(inputs.ref_fasta.path) \
+            OUTPUT=$(inputs.output_bam_prefix) \
+            ASSUME_SORTED=true \
+            PROGRAM=null \
+            PROGRAM=CollectAlignmentSummaryMetrics \
+            $(inputs.collect_gc_bias_metrics ? "PROGRAM="CollectGcBiasMetrics"" : "") \
```

Comment:
```suggestion
            $(inputs.collect_gc_bias_metrics ? 'PROGRAM="CollectGcBiasMetrics"' : "") \
```
like in the original WDL https://github.com/gatk-workflows/gatk4-exome-analysis-pipeline/blob/3c43ebd645759c0895539ea4231dad13bb1ae363/tasks/Qc.wdl#L174